### PR TITLE
Add Layout page, restructure table of contents, other miscellaneous changes

### PR DESCRIPTION
**Pull request by jtamiace**
## Reference

https://docs.google.com/document/d/1xoU5sBy2CR7IBCYoIJxXF69WZDrKLysUDMg5kwVbyQ0/edit#heading=h.ur4m0c79z9lu

## Changes

- Adds Layout page
- Changes "Design patterns" section in table of contents to "Foundation," "Components," and "Patterns." Also auto-sorts pages for each of these new sections
- Removes link to core values page and reference to "Design for Everyone" since this will soon not be on the website
- Adds Khang's name to list of consultants on Home page

## Library to do

- Make KGrid's breakpoint ranges match Vuetify's for consistency

### CODE REVIEW

File: docs/pages/layout/index.vue [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+<style lang="scss" scoped>
+
+  td,
+  th {
+    padding: 8px;
+    text-align: left;
+    vertical-align: top;
+  }
+
+</style>
```
Comment (nucleogenesis) [inline docs/pages/layout/index.vue:154]: Not sure if it's important but Github made a point to mention that there is no EOL newline at the end of the file. I thought that our linter typically complained about this.

File: docs/pages/index.vue [hunk 2]
```diff
@@ -64,7 +64,7 @@
         In the meantime, we welcome direct feedback and contributions! It's important that the Design System works for you, so please help shape it.
       </p>
       <ul>
-        <li>Share ideas or concerns with Devon, Jessica, and Jacob directly</li>
+        <li>Share ideas or concerns with Devon, Jessica, Khang, and Jacob directly</li>
```
Comment (nucleogenesis) [inline docs/pages/index.vue:67]: I wonder if our names should link to some way to reach out to us. Maybe our Github tags or emails?

This makes sense for internal use but community users won't know who those people are.

File: docs/pages/layout/index.vue (hunk 1 above)
Comment (indirectlylit) [reply docs/pages/layout/index.vue]: I believe in Kolibri that's handled by some pre-commit hooks, not `kolibri-tools`

File: docs/pages/index.vue (hunk 2 above)
Comment (indirectlylit) [reply docs/pages/index.vue:67]:
```suggestion
        <li>Share ideas or concerns with Devon, Jessica, Khang, and Jacob directly, or file issues in this repo.</li>
```

File: docs/pages/layout/index.vue [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+        <li>Padding: 16px (small screens), 24px (medium and large screens)</li>
+      </ul>
+      <h3>
+        Max width
+      </h3>
+      <p>
+        If a page container contains text, set its maximum width to 1000px in order to aid readability on larger screens.
+      </p>
+      <h3>
+        Dividers
```
Comment (indirectlylit) [inline docs/pages/layout/index.vue:35]: These are often termed "horizontal rules" - not sure if we want to adopt that terminology

File: docs/pages/layout/index.vue [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+      <h3>
+        Specs
+      </h3>
+      <ul>
+        <li>Width: 1px</li>
+        <li>Color: tokens.fineline (#dedede)</li>
+      </ul>
+    </DocsPageSection>
+
+    <DocsPageSection title="Responsive Grids" anchor="#grids">
```
Comment (indirectlylit) [inline docs/pages/layout/index.vue:53]: Assuming we're using our own writing guidelines, headers should be `Sentence case`

```suggestion
    <DocsPageSection title="Responsive grids" anchor="#grids">
```

File: docs/pages/layout/index.vue [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+      <p>
+        Each breakpoint range has default: number of columns, gutter width, margin, and padding. These defaults can be changed according to exceptional needs.
+      </p>
+      <table>
+        <tr>
+          <th>Range</th>
+          <th><DocsExternalLink href="https://v15.vuetifyjs.com/en/framework/grid" text="Vuetify" /></th>
+          <th>KGrid</th>
+          <th>Columns</th>
+          <th>Margins & Gutters</th>
```
Comment (indirectlylit) [inline docs/pages/layout/index.vue:85]:
```suggestion
          <th>Margins &amp; Gutters</th>
```

File: docs/pages/layout/index.vue [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+      <p>
+        Use sparingly and reserve them for situations where white space does not properly separate elements or groups of elements, such as a table, compact list items, or list sections.
+      </p>
+      <img src="./dividers.png">
+      <h3>
+        Specs
+      </h3>
+      <ul>
+        <li>Width: 1px</li>
+        <li>Color: tokens.fineline (#dedede)</li>
```
Comment (indirectlylit) [inline docs/pages/layout/index.vue:49]:
```suggestion
        <li>
          Color: <DocsInternalLink href="/colors#tokens-fineLine" code>
            tokens.fineline
          </DocsInternalLink> (<code>#dedede</code>)
        </li>
```

File: docs/pages/layout/index.vue (hunk 3 above)
Comment (jtamiace) [reply docs/pages/layout/index.vue:39]: Material uses the term "Dividers" which is why I went for it here https://material.io/components/dividers

File: docs/tableOfContents.js [hunk 7]
```diff
@@ -81,36 +74,56 @@ export default [
...
+    title: 'Patterns',
+    autoSort: true,
+    pages: [
+      new Page({
+        path: '/forms',
+        title: 'Forms',
+        disabled: true,
+      }),
+    ],
+  }),
```
Comment (indirectlylit) [inline docs/tableOfContents.js:123]: maybe 'Patterns' should be above 'Components'?

Then the ordering of sections would go from higher to lower levels of abstraction

Comment (jtamiace) [reply docs/tableOfContents.js]: Done, makes sense

**Pull request merged**